src/views/main.js

```diff
@@ -22,7 +23,13 @@ const mainLayout = data => `
     <link rel="preload" href="/fonts/Metropolis-Bold.woff2" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="/fonts/Inter-Regular-latin.woff2" as="font" type="font/woff2" crossorigin>
     <link rel='stylesheet' href='/css/index.css' type='text/css'>
-    <link rel='stylesheet' href='/css/partials/${data.partial.name}.css' type='text/css'>
+    ${hasPartials(data)
+      ? data.partials.map(partial => (
+          `<link rel='stylesheet' href='/css/partials/${partial.name}.css' type='text/css'>`
+        )).join()
+      : ''
+    }
```

Comment:
Load the necessary stylesheets for the partials that we would like to use.

Reply:
Not blocking, but maybe we should consider doing this for JS too... Am I correct in understanding all the client-side JS gets loaded by all pages currently?

Reply:
@rhelmer that is correct – I'm keeping an eye on JS bundle size and it's still quite low: 4.35 kB transfered (for non-dev environments).  Our CSS tends to be a bit heavier, so splitting those files per-page is currently providing a better benefit.  Will keep _monitoring_!